webpack/assets/javascripts/react_app/components/common/EmptyState/DefaultEmptyState.js

```diff
@@ -1,43 +1,58 @@
 import React from 'react';
+import { Button } from '@patternfly/react-core';
 import EmptyStatePattern from './EmptyStatePattern';
-import PrimaryActionButton from './EmptyStatePrimaryActionButton';
-import SecondaryActionButtons from './EmptyStateSecondaryActionButtons';
 import { defaultEmptyStatePropTypes } from './EmptyStatePropTypes';
-import { translate as __ } from '../../../common/I18n';
 
 const DefaultEmptyState = props => {
   const {
     icon,
     iconType,
     header,
     description,
-    documentation: {
-      url,
-      label = __('For more information please see'),
-      buttonLabel = __('Documentation'),
-    } = {},
+    documentation,
     action,
     secondaryActions,
   } = props;
 
-  const documentationBlock = url ? (
-    <React.Fragment>
-      {label}{' '}
-      <a href={url} target="_blank" rel="noopener noreferrer">
-        {buttonLabel}
-      </a>
-    </React.Fragment>
+  const actionButtonClickHandler = ({ url, onClick }) => {
+    onClick && onClick();
```

Comment:
```suggestion
  const actionButtonClickHandler = ({ url, onClick = noop }) => {
    onClick();
```
This way we can trigger the function with no fear

Reply:
This doesn't work in the expected manner :(